samples/snippets/get_model_evaluation_tabular_forecasting_sample.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Google LLC
```

Comment:
Same as corresponding classification sample.

Reply:
I think we don’t really need to have sample variants for get and list methods. I see that we have them for some other kinds of AutoML services and perhaps those need to be removed (in a separate PR).

Reply:
Sure!